changelog/_unreleased/2023-04-19-add-support-for-active-flag-of-user-entity.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Add support for active flag of user entity
+issue: NEXT-26196
+author: Timo Helmke
+author_email: t.helmke@kellerkinder.de
+author_github: t2oh4e
+---
+# Core
+*
```

Comment:
would you mind changing this to something meaningful? 😁

Reply:
Wooopsie 😅  Changed.